playbooks/code-review.md

```diff
@@ -2,3 +2,45 @@
 title: Code reviews
 description: How our code review process works 
 ---
+
+Reviewing other people's code is a responsibility of every single developer. Instead of viewing it as a hassle, it should be looked upon as a necessary process in order to keep the code clean and consistent.
+It is also an opportunity to learn something new.
+
+Rules on..
+
+## How a Code Review process should look like
+
+..are written below.
+
+### Terms:
+ - PR - Pull Request.
+ - PR Creator - a person who initiated the PR.
+ - Reviewer - a person assigned to do a PR code review.
+
+### Rules:
+1. Every PR has to be reviewed, with the exception of fix/hotfix PR-s that can be merged by the code owner on his own responsibility.
+2. When creating a PR, you should select **EXACTLY ONE** person as a reviewer. This person has to be someone who is working at the moment (not on vacation). 
+If there is someone else who should see this PR, he/she should be assigned as Assignee on this PR.
+	Reviewer Priority list:
+	1. Most senior developer on the project (excluding you).
+	2. Anyone else on the project.
+	3. Anyone else. (predetermined as a backup)
+3. After a reviewer is selected, responsibility for this PR is transfered to him/her. Reviewer has options to:
+	1. Do a Code Review and Approve PR.
+	2. Do a Code Review and Request Changes on the PR.
+	3. Return a response that he/she will be able to do Code Review some time later.
+	d. Return a response that he/she wont be able to do a Code Review.
+4. After a reviewer has responded (3a,3b,3c,3d), the PR creator has to do one of the following:
+	1. (3a) Merge PR.
+	2. (3b) Make changes.
+	3. (3c,3d) find someone else for the Code Review.
+5. Repeat 3. and 4. untill the PR is merged.
+
+### Important Notes:
```

Comment:
I would also love to see a list "what to think about when code reviewing someones code"  and a link to a example of a great code review

Reply:
I agree with Tony.

Since these are guidelines for PR author, we could add guidelines for PR reviewer.
@markopaden What do you think?

Reply:
I agree, but as I said on the meeting, I think it should be made in a separate document. Actual reviewing of the code is only one step in the process that deserves it's own section.

Reply:
Do you mind then creating that document so it gets in the "up for grabs" list ?